fetch.bs

```diff
@@ -4648,21 +4648,108 @@ steps:
       <p class=note>The `<code>GET</code>` <a for=/>method</a> restriction serves no useful purpose
       other than being interoperable.
 
-     <li><p>Let <var>bodyWithType</var> be the result of <a for=BodyInit>safely extracting</a>
-     <var>blobURLEntry</var>'s <a for="blob URL entry">object</a>.
+     <li><p>Set <var>response</var> to a new <a for=/>response</a>.
 
-     <li><p>Let <var>body</var> be <var>bodyWithType</var>'s <a for="body with type">body</a>.
+     <li><p>If <var>request</var>'s <a for=response>header list</a> contains `<code>Range</code>`,
+     then set <var>response</var>'s <a for=response>range-requested flag</a>.
 
-     <li><p>Let <var>length</var> be <var>body</var>'s <a for=body>length</a>,
+     <li><p>Let <var>fullLength</var> be <var>blobURLEntry</var>'s
+     <a for="blob URL entry">object</a>'s <a for=Blob data-link-type=attribute>size</a>,
```

Comment:
We can use `{{Blob/size}}` as we do elsewhere in this document. Same for type.

Reply:
No problem! Just for future reference, what is the benefit of `{{Blob/size}}` and how does it differ from `<a for=Blob data-link-type=attribute>size</a>`? Both create links to https://w3c.github.io/FileAPI/#dfn-size for me.

Reply:
Does the latter include also result in `<code>` being included? Anyway, the former is a shorthand that should do the right thing and it's useful to do it consistently within a document so you can easily find all instances.

Now having said all that, typically you would never want to reference an IDL attribute as it's the public API. The problem here is that `Blob` still doesn't define its internal model.

Reply:
> Does the latter include also result in <code> being included?

Ah! No it does not.

> Anyway, the former is a shorthand that should do the right thing and it's useful to do it consistently within a document so you can easily find all instances.

This makes sense.